app/controllers/admin/users_controller.rb

```diff
@@ -272,7 +272,7 @@ def set_related_reactions
       @related_vomit_reactions =
         Reaction.where(reactable_type: "Comment", reactable_id: user_comment_ids, category: "vomit")
           .or(Reaction.where(reactable_type: "Article", reactable_id: user_article_ids, category: "vomit"))
-          .or(Reaction.where(reactable_type: "User", user_id: @user.id, category: "vomit"))
+          .or(Reaction.where(reactable_type: "User", reactable_id: @user.id, category: "vomit"))
```

Comment:
rather than reactions by the user, show reactions to the user (by others, a report).